src/som/interpreter/actors/Actor.java

```diff
@@ -241,11 +269,13 @@ protected void processCurrentMessages(final ActorProcessingThread currentThread,
 
       try {
         execute(firstMessage, currentThread, dbg, -1);
+        stopNextMessage = firstMessage.isStepOver();
```

Comment:
this should only be done when debugging is enabled, no?
and since you do it also in the loop below, should it be part of the `execute()` method?